tests/test_speed.py

```diff
@@ -25,7 +25,7 @@
             "from onecodex.viz import VizPCAMixin",
             {
                 "onecodex": 0.25,
-                "onecodex.viz": 0.20,
+                "onecodex.viz": 0.25,
```

Comment:
This is unrelated to the changes in this PR. I've noticed this test fails quite frequently, let's see if bumping the timeout helps.